templates/cluster/openstack-standalone-cp/values.yaml

```diff
@@ -0,0 +1,89 @@
+# cluster parameters
+controlPlaneNumber: 3
+workersNumber: 2
+
+clusterNetwork:
+  pods:
+    cidrBlocks:
+    - "10.244.0.0/16"
+  services:
+    cidrBlocks:
+    - "10.96.0.0/12"
+  serviceDomain: "cluster.local"
+
+# openstack cluster parameters
+openstack:
+  authURL: "" # Keystone URL
+  region: "" # OpenStack region
+  username: "" # OpenStack username
+  password: "" # OpenStack password
+  useApplicationCredentials: false # set to true if using application credentials instead of username/password
+  applicationCredentialID: "" # OpenStack application credential ID (if applicable)
+  applicationCredentialSecret: "" # OpenStack application credential secret (if applicable)
```

Comment:
It looks like the whole openstack section makes no sense for the template

Reply:
Right, since I removed it from k0scontrolplane, I will remove it here too.